tests/pytests/test_issues.py

```diff
@@ -888,3 +888,35 @@ def test_mod5778_add_new_shard_to_cluster(env):
     # cleanup
     new_instance.kill()
     os.remove('nodes.conf')
+
+
+@skip(cluster=True)
```

Comment:
Its messy since we cannot use `ft.config` with coordinator and the output of `ft.profile` is different. So I don't think that there is an added value for testing with a coordinator given these constraints...